### PR TITLE
KLBP-249: Association step 1 implementation.

### PR DESCRIPTION
**Pull request by andreir-suciu**

### CODE REVIEW

File: src/__tests__/pages/steps/Step2PrivateCompany.test.tsx [hunk 1]
```diff
@@ -688,7 +688,7 @@ test('The form in edit mode can be canceled', async () => {
       screen.getByText('Cliquer sur cette carte pour corriger la personne.')
     ).toBeInTheDocument()
   })
-})
+}, 10000)
```
Comment (mariuspop86) [inline src/__tests__/pages/steps/Step2PrivateCompany.test.tsx:691]: Try to avoid setting this timeout

**Review by ioan-iacob-netea — APPROVED** [review]

File: src/components/Journeys/Association/Step1/Step1UploadTooltips/JDDDocumentTooltipContent.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <>
+      <DynamicFormattedMessage
+        id="association.step1.button3.tooltip.title"
+        tag="strong"
+        className="mb-2"
+      />
+      <ul className="pl-5 list-disc">
+        <li className="mb-2">
+          <DynamicFormattedMessage
+            id="association.step1.button3.tooltip.text1"
```
Comment (mariuspop86) [inline src/components/Journeys/Association/Step1/Step1UploadTooltips/JDDDocumentTooltipContent.tsx:15]:
```
const tooltipsLIst = ['association.step1.button3.tooltip.text1','association.step1.button3.tooltip.text2','association.step1.button3.tooltip.text3', 'association.step1.button3.tooltip.text4']
return (
  <>
    <DynamicFormattedMessage
        id="association.step1.button3.tooltip.title"
        tag="strong"
        className="mb-2"
      />
      <ul className="pl-5 list-disc">
   {tooltipsLIst.map(id => 
       <li className="mb-2">
          <DynamicFormattedMessage
            id={id}
            tag="p"
          />
        </li>
```

File: src/components/Journeys/Association/Step1/SirenRnaInput.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <>
+      <div className="flex flex-wrap mb-6 md:mb-0">
+        <div className="flex items-center w-1/2">
+          <DynamicFormattedMessage
+            className="text-2xl leading-tight text-black text-opacity-80 mr-4"
+            id={`association.step1.${isSiren ? 'siren' : 'rna'}.label`}
+            tag="span"
+          />
+          <span className="text-warning-DEFAULT">**</span>
```
Comment (iuliudascalu) [inline src/components/Journeys/Association/Step1/SirenRnaInput.tsx:23]: I think will be better if  use an translation for the `**` for exemple on private company we have for a single `*` the `"mandatory.field.symbol`.

File: src/components/Journeys/Association/Step1/Step1UploadLabels/Step1AnnualBudgetLabel.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    <div className="flex items-start">
+      <div>
+        {labelId && (
+          <DynamicFormattedMessage
+            id={labelId}
+            tag="span"
+            className="text-2xl leading-tight"
+          />
+        )}
+        {isRequired && <span className="text-danger-DEFAULT"> *</span>}
```
Comment (iuliudascalu) [inline src/components/Journeys/Association/Step1/Step1UploadLabels/Step1AnnualBudgetLabel.tsx:22]: Same as above, we should use the `mandatory.field.symbol` here, i think we can use that after the private company journey is merged into recette or we can add the translation in fr.json one more time and we will fix the conflicts (if we'll have) after that.

File: src/components/Journeys/Association/Step1/Step1UploadLabels/Step1FatcaLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    <div className="flex items-start">
+      <div>
+        {labelId && (
+          <DynamicFormattedMessage
+            id={labelId}
+            tag="span"
+            className="text-2xl leading-tight"
+          />
+        )}
+        {isRequired && <span className="text-danger-DEFAULT"> *</span>}
```
Comment (iuliudascalu) [inline src/components/Journeys/Association/Step1/Step1UploadLabels/Step1FatcaLabel.tsx:25]: Same as above :D

File: src/components/Steps/Step2Content.tsx [hunk 6]
```diff
@@ -11,10 +11,17 @@ const Step2Content = () => {
...
+  const {
+    INDIVIDUAL_COMPANY,
+    ASSOCIATION_NON_SIRENE,
+    PRIVATE_COMPANY,
+    ASSOCIATION_SIRENE,
+  } = CLIENT_TYPOLOGY
 
   switch (typologie_client) {
     case INDIVIDUAL_COMPANY:
+    case ASSOCIATION_SIRENE:
```
Comment (iuliudascalu) [inline src/components/Steps/Step2Content.tsx:23]: Are you sure this is ok ? the association sirene and non sirene should load the same component as individual company?

Comment (andreir-suciu) [reply src/components/Steps/Step2Content.tsx:23]: This is the step 1 implementation. I added step 2  from EI just for testing purposes. This is updated in step 2 implementation
https://github.com/beprems/KYB-lbp-front/pull/234

**Review by andreibuhus — APPROVED** [review]

File: src/enums/journeys.ts [hunk 7]
```diff
@@ -7,9 +7,9 @@ export const STEP_3_ID = '2'
 
 export const CLIENT_TYPOLOGY = {
   INDIVIDUAL_COMPANY: '3',
-  NON_SIRENEE_ASSOCIATION: '2',
+  ASSOCIATION_NON_SIRENE: '2',
```
Comment (mariuspop86) [inline src/enums/journeys.ts:10]: This should be in alphabetic order

File: src/components/Journeys/Association/Step1/Step1UploadLabels/Step1AnnualBudgetLabel.tsx (hunk 4 above)
Comment (andreir-suciu) [reply src/components/Journeys/Association/Step1/Step1UploadLabels/Step1AnnualBudgetLabel.tsx]: There should be no conflicts as I updated association feature branch with last version of SDP feature branch. :dancers:

**Review by mariuspop86 — APPROVED** [review]

**Review by iuliudascalu — APPROVED** [review]

**Pull request merged**